### DISCUSSION TITLE
Fancybox, open images zoomed in

### DISCUSSION THREAD

**Discussion by addedlovely** [fancyapps/ui#367] [Q&A] [answered]
Hello.

Is it possible to open images in a zoomed in state using Fancybox. Have been trying various options to no avail:
https://fancyapps.com/playground/3UP

Image settings below do zoom the image, but only after an initial click to start the zoom, ideally would like to have the image load in a zoomed in state.
```
Image: {
  Panzoom: {
    baseScale : 2,
    maxScale: 3,
    panOnlyZoomed: false
  },
},
```

If I try and call the zoom method on the 'done' event, it works on the first image, then fails on the second, with TypeError: undefined is not an object (evaluating 'Fancybox.getInstance().getSlide().Panzoom.zoomIn')

```
on: {
  done: (fancybox) => {
    Fancybox.getInstance().getSlide().Panzoom.zoomIn();
  },
}

```

Thanks!

Ian.

**Comment by fancyapps** [comment, marked as answer]
Hi,

Sorry, v4 is not optimised for that. It can work, but it will not look great. However, fancybox5 is around the corner where there will be an option for that (e.g., to start images zoomed-in).

**Reply by addedlovely** [reply]
No problem, thanks for updating - will keep an eye out for V5!

**Reply by b-s-j** [reply]
omg I haven't full understand v4 and yet there is v5 coming out T_T 👯
